### DISCUSSION TITLE
Are you open to pull request? I would love to include my open source project

### DISCUSSION THREAD

**Discussion by SquirrelDeveloper** [tteck/Proxmox#3889] [Ideas]
Hi,
Just checking if you are open to PRs, I would like to include my open source project, [Squirrel Servers Manager](https://squirrelserversmanager.io/)
Regards

**Comment by MickLesk** [comment]
Hi,

If you follow the guidelines and code as specified, there’s no reason not to move forward with it. Alternatively, you can create a script template based on your best judgment, and I can review it, though tteck will probably give it a final check.

I find your project pretty interesting and could see it working well as an LXC setup. Just keep in mind, we don’t "support" Docker—we rely on native repos and installations.

https://github.com/tteck/Proxmox/blob/main/.github%2FCONTRIBUTING.md
